runner/commander.go

```diff
@@ -96,6 +99,70 @@ func (c Commander) Run() op.Op {
 	}
 }
 
+type parsedCommand struct {
+	cmd  string
+	args []string
+	err  error
+}
+
+func parseCommand(command string) parsedCommand {
```

Comment:
```suggestion
func parseCommand(command string) (parsedCommand, error) {
```

Make sure to mutlireturn the error as a part of the fn signature to keep the api idiomatic. You always want to return the error itself for immediate handling. It's assumed that the return value isn't usable if err != nil, and this could create a gotcha in the future if the return value is assumed to always be ok by a consumer.

Reply:
Good call!